rules/integrations/google_workspace/defense_evasion_domain_added_to_google_workspace_trusted_domains.toml

```
@@ -24,7 +24,47 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Domain Added to Google Workspace Trusted Domains"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Domain Added to Google Workspace Trusted Domains
+
+Organizations use trusted domains in Google Workspace to give external users access to resources.
+
+A threat actor with administrative privileges may be able to add a malicious domain to the trusted domain list. Based on the configuration, potentially sensitive resources may be exposed or accessible by an unintended third-party.
+
+This rule detects when a third-party domain is added to the list of trusted domains in Google Workspace.
+
+#### Possible investigation steps
+
+- Identify the associated user accounts by reviewing `user.name` or `user.email` fields in the alert.
+- After identifying the user, verify if the user should have administrative privileges to add external domains.
+- Check the `google_workspace.admin.domain.name` field to find the newly added domain.
+- Use reputational services, such as VirusTotal, for the trusted domain's third-party intelligence reputation.
+- Filter your data Discover. Create a filter where `event.dataset` is `google_workspace.drive` and `google_workspace.drive.file.owner.email` is being compared to `user.email`.
```

```suggestion
- Filter your data. Create a filter where `event.dataset` is `google_workspace.drive` and `google_workspace.drive.file.owner.email` is being compared to `user.email`.
```
Seems like an extra word?